utils/centos7-ci.sh

```diff
@@ -20,36 +20,29 @@
 
 set -ex
 
-install_dependencies() {
-    export PATH=/usr/local/openresty-debug/nginx/sbin:/usr/local/openresty-debug/bin:$PATH
-    yum install -y wget tar gcc automake autoconf libtool make
-    wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-    rpm -ivh epel-release-latest-7.noarch.rpm
-    yum install -y curl git luarocks lua-devel which
-    yum install -y yum-utils && yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
-    yum install -y openresty-debug
-    wget https://github.com/etcd-io/etcd/releases/download/v3.4.0/etcd-v3.4.0-linux-amd64.tar.gz
-    tar xf etcd-v3.4.0-linux-amd64.tar.gz
-    cp /etcd-v3.4.0-linux-amd64/etcdctl /usr/local/bin/
-    rm -rf etcd-v3.4.0-linux-amd64
-    yum install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl
-    cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)
-    mkdir build-cache 
-    wget https://github.com/iresty/grpc_server_example/releases/download/20200901/grpc_server_example-amd64.tar.gz
-    tar -xvf grpc_server_example-amd64.tar.gz 
-    mv grpc_server_example build-cache/
-    git clone https://github.com/iresty/grpc_server_example.git grpc_server_example 
-    cd grpc_server_example/ && mv proto/ ../build-cache/ && cd ..
-    ./build-cache/grpc_server_example > grpc_server_example.log 2>&1 || (cat grpc_server_example.log && exit 1)&
-    sleep 3
-    cd apisix
-    make deps
-    make init
-    git clone https://github.com/iresty/test-nginx.git test-nginx
-}
 
-run_case() {
-    export PATH=/usr/local/openresty-debug/nginx/sbin:/usr/local/openresty-debug/bin:$PATH
-    cd apisix
-    prove -Itest-nginx/lib -I./ -r t/
-}
\ No newline at end of file
+export PATH=/usr/local/openresty-debug/nginx/sbin:/usr/local/openresty-debug/bin:$PATH
```

Comment:
wow, please choose a better style, the current style is not easy for reading.

eg: add more blank line and need some comments